### PR TITLE
Added the capital L

### PR DESCRIPTION
**Pull request by IlijazM**
This construction should return the capital L. Although I wasn't able to implement it in your code, so it returns an error somehow, maybe you can have a look at this and perhaps implement it properly.

### CODE REVIEW

**Comment by frobinsonj** [comment]
Could never think of a way to get L, I like this :)

`(""+Object["entries"](Function("return Error")()))[10]` will work :)

**Comment by IlijazM** [comment]
Ok, now everything should work. Have fun with it and keep expanding your awesome projects ;)

File: jsfuck.js [hunk 1]
```diff
@@ -63,15 +63,15 @@
...
+    'L':   '(""+Object["entries"](Function("return Error")()))[10]',
     'M':   '(true+Function("return Date")()())[30]',
     'N':   '(NaN+"")[0]',
     'O':   '(+[]+Object)[10]',
     'P':   USE_CHAR_CODE,
     'Q':   USE_CHAR_CODE,
     'R':   '(+[]+RegExp)[10]',
     'S':   '(+[]+String)[10]',
-    'T':   '(NaN+Function("return Date")()())[30]',
+    'T':   '(""+Object["entries"](Function("return Error")()))[5]',
```
Comment (frobinsonj) [inline jsfuck.js:74]: You should use `(false+Object["entries"](Function("return Error")()))[10]` here. It actually saves 15 characters by avoiding using the number 5. "10" is much cheaper, it's just "1" + "0".

Comment (IlijazM) [reply jsfuck.js]: Ou, of course, that seems reasonable. Here you go.

**Comment by hazzik** [comment]
This, unfortunately, does not work in Firefox and PhantomJS

**Comment by aemkei** [comment]
As FF is not supported, I'll close this.

**Pull request closed without merging**